src/Symfony/Bundle/FrameworkBundle/Test/TestContainer.php

```diff
@@ -97,15 +97,15 @@ public function has($id)
     /**
      * {@inheritdoc}
      */
-    public function get($id, $invalidBehavior = /* self::EXCEPTION_ON_INVALID_REFERENCE */ 1)
+    public function get($id, int $invalidBehavior = /* self::EXCEPTION_ON_INVALID_REFERENCE */ 1)
```

Comment:
can't we add the type to the $id also?

Reply:
No, because the upstream PSR interface doesn't have it.